stylesheets/layout/layout.css

```diff
@@ -159,8 +181,6 @@
   color: var(--white-color);
   background-color: var(--secondary-color);
   border-radius: var(--border-rd-md);
-  margin-top: 0.2rem;
```

Comment:
try and use px if values are too small.

Reply:
thank you for the suggestion, I'll keep this in mind next time on